sparkplug/executor.py

```diff
@@ -2,7 +2,6 @@
 import signal
 import time
 import multiprocessing
-from builtins import range
```

Comment:
Does that mess with Py2/Py3 compat?

Reply:
From what I can tell, no. Both have a range function, and for what we use it for, it behaves the same.
https://docs.python.org/2/library/functions.html#range
```
Python 2.7.10 (default, Oct  6 2017, 22:29:07) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> test = [i for i in range(3)]
>>> test
[0, 1, 2]
```
https://docs.python.org/3/library/functions.html#func-range
```
Python 3.7.2 (default, Jan 13 2019, 12:50:15) 
[Clang 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> test = [i for i in range(3)]
>>> test
[0, 1, 2]
```